Robot_Engine/Convert/Private Helpers/Load.cs

```diff
@@ -128,23 +133,48 @@ public static void RobotLoad(this PointLoad load, RobotSimpleCase sCase, RobotGr
 
         public static void RobotLoad(this BarUniformlyDistributedLoad load, RobotSimpleCase sCase, RobotGroupServer rGroupServer)
         {
-            IRobotLoadRecord loadRecordForce = sCase.Records.Create(IRobotLoadRecordType.I_LRT_BAR_UNIFORM);
-            loadRecordForce.Objects.FromText(load.CreateIdListOrGroupName(rGroupServer));
-            loadRecordForce.SetValue((short)IRobotBarUniformRecordValues.I_BURV_PX, load.Force.X);
-            loadRecordForce.SetValue((short)IRobotBarUniformRecordValues.I_BURV_PY, load.Force.Y);
-            loadRecordForce.SetValue((short)IRobotBarUniformRecordValues.I_BURV_PZ, load.Force.Z);
-
-            IRobotLoadRecord loadRecordMoment = sCase.Records.Create(IRobotLoadRecordType.I_LRT_BAR_MOMENT_DISTRIBUTED);
-            loadRecordMoment.Objects.FromText(load.CreateIdListOrGroupName(rGroupServer));
-            loadRecordMoment.SetValue((short)IRobotBarMomentDistributedRecordValues.I_BMDRV_MX, load.Moment.X);
-            loadRecordMoment.SetValue((short)IRobotBarMomentDistributedRecordValues.I_BMDRV_MY, load.Moment.Y);
-            loadRecordMoment.SetValue((short)IRobotBarMomentDistributedRecordValues.I_BMDRV_MZ, load.Moment.Z);
+            if (load.Force.Length() == 0 && load.Moment.Length() == 0)
+            {
+                Engine.Reflection.Compute.RecordError("Zero forces and moments are not pushed to Robot");
+                return;
+            }
+
+            if (load.Force.Length() != 0)
+            {
+                IRobotLoadRecord loadRecordForce = sCase.Records.Create(IRobotLoadRecordType.I_LRT_BAR_UNIFORM);
+                loadRecordForce.Objects.FromText(load.CreateIdListOrGroupName(rGroupServer));
+                loadRecordForce.SetValue((short)IRobotBarUniformRecordValues.I_BURV_PX, load.Force.X);
+                loadRecordForce.SetValue((short)IRobotBarUniformRecordValues.I_BURV_PY, load.Force.Y);
+                loadRecordForce.SetValue((short)IRobotBarUniformRecordValues.I_BURV_PZ, load.Force.Z);
+            }
+            else
```

Comment:
Dont think we need/want this error raised here. 

As the first check has passed, this load will have either moment or force, so something will get pushed.

Now if you try to apply only a UDL-moment you will get errors about no force.